GoogleForCreators/web-stories-wp
# I18N: Replace `@wordpress/i18n` with Tannin

## Context



See #8577 for context

## Summary



Replace `@wordpress/i18n` with Tannin, the underlying library used by it.

## Relevant Technical Choices



## To-do



- [x] Ensure string extraction still works (i18n lint should pass)
- [x] Reduce bundle size if possible
- [x] Ensure  that `wp-i18n` script is not unnecessarily enqueued
- [x] Fix inline script calls for all scripts _and_ chunks

## User-facing changes



N/A

## Testing Instructions






This PR can be tested by following these steps:

1. Switch your site/user language to something like Portuguese (Brazil)
2. Go to Dashboard -> Updates to ensure all translations are installed
3. Go to the story dashboard and the story editor and verify that the majority of strings are translated.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8577
